packages/playout-gateway/src/configManifest.ts

```diff
@@ -363,8 +363,9 @@ const PLAYOUT_SUBDEVICE_CONFIG: ImplementedSubDeviceConfig = {
 			type: ConfigManifestEntryType.INT,
 		},
 		{
-			id: 'options.showID',
-			name: 'Show ID',
+			id: 'options.showDirectoryPath',
+			name: '(Optional) Show Directory Path',
```

Comment:
Isn't this only optional because our blueprints graps the value from somewhere else?
For NRK wouldn't this still be required?